LayoutTests/http/tests/webgpu/webgpu/idl/exposed.http-expected.txt

```diff
@@ -1 +1,5 @@
-(Populate me when we're ready to investigate this test)
+layer at (0,0) size 800x600
+  RenderView at (0,0) size 800x600
+layer at (0,0) size 800x8
+  RenderBlock {HTML} at (0,0) size 800x8
+    RenderBody {BODY} at (8,8) size 784x0
```

Comment:
Is this actually passing?

Reply:
Can we figure out why this isn't turning into a text test but is instead a render tree test?